workers/caching/metaphysics-cdn.ts

```diff
@@ -69,6 +62,23 @@ export default {
     }
 
     try {
+      if (
+        request.method.toUpperCase() === "OPTIONS" &&
+        request.headers.get("origin")?.match(/\.artsy\.net$/)
+      ) {
+        // Handle preflight requests
+        return new Response(null, {
+          status: 204,
+          headers: {
+            "Access-Control-Allow-Origin": "*",
+            "Access-Control-Allow-Methods": "GET,POST,OPTIONS",
+            "Access-Control-Allow-Headers":
+              "cache-control,content-type,x-access-token,x-original-session-id,x-relay-cache-config,x-relay-cache-path,x-timezone,x-user-id",
```

Comment:
we don't need those `x-relay-cache` headers - they only made sense in the proxy - and actually after https://github.com/artsy/force/pull/14569 we shouldn't be sending then anymore